packages/customize-widgets/src/components/sidebar-block-editor/use-sidebar-block-editor.js

```diff
@@ -0,0 +1,240 @@
+/**
+ * External dependencies
+ */
+import { invert, omit, keyBy, isEqual } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { serialize, parse, createBlock } from '@wordpress/blocks';
+import { useState, useEffect, useCallback, useRef } from '@wordpress/element';
+
+function blockToWidget( block, existingWidget = null ) {
```

Comment:
These functions already exist in `edit-widgets`. Noting that we'll want to do some DRYIng up eventually. Again, not right now though.